src/odemis/driver/rigol.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, name, role, host, port=DEFAULT_PORT, channel=1, limits=(0, 10
         self._phase_shift = 0
         self._dc_bias = self._amplitude_pp / 2 + lo
 
-        self._duty_cycle = 50 #%
```

Comment:
This shouldn't be part of the commit. Actually the change is only needed for the testing, so it can be completely dropped.